src/redux/uniqueTokens.ts

```diff
@@ -69,9 +220,20 @@ export const uniqueTokensRefreshState = () => async (dispatch, getState) => {
   dispatch(fetchUniqueTokens());
 };
 
-export const fetchUniqueTokens = showcaseAddress => async (
-  dispatch,
-  getState
+/**
+ * Fetches unique tokens or showcased unique tokens via API, updates state,
+ * and saves to local storage.
+ *
+ * @param showcaseAddress The showcase address to use for fetching, or no
+ * address to fetch all unique tokens.
+ */
+export const fetchUniqueTokens = (showcaseAddress?: string) => async (
+  dispatch: ThunkDispatch<
+    AppState,
+    unknown,
+    UniqueTokensGetAction | UniqueTokensClearStateShowcaseAction
+  >,
+  getState: AppGetState
 ) => {
   dispatch({
     showcase: !!showcaseAddress,
```

Comment:
starting at line 242:
```
if (showcaseAddress) {
    dispatch({
      showcase: !!showcaseAddress,
      type: UNIQUE_TOKENS_CLEAR_STATE_SHOWCASE,
    });
  }```

the dispatch does not need to pass along `showcase: !!showcaseAddress` as it is unused

Reply:
@jinchung Okay, I removed that line, thanks!